### PR TITLE
Add error tag to ext/tags.py

### PR DESCRIPTION
**Pull request by oibe**

### CODE REVIEW

**Comment by oibe** [comment]
fixes https://github.com/opentracing/opentracing-python/issues/35

File: opentracing/ext/tags.py [hunk 1]
```diff
@@ -37,6 +37,11 @@
 SPAN_KIND_RPC_SERVER = 'server'
 
 # ---------------------------------------------------------------------------
+# ERROR indicates whether a Span ended in an error state.
+# ---------------------------------------------------------------------------
+ERROR = 'error'
```
Comment (bhs) [inline opentracing/ext/tags.py]: thanks!

in the "no good deed goes unpunished" dept, would you be willing to add the other missing tags from https://github.com/opentracing/specification/blob/master/data_conventions.yaml? Esp the new database ones.

Comment (oibe) [inline opentracing/ext/tags.py]: Sorry just saw this.  However the link above is broken.  Can you re-link me?

Comment (bhs) [inline opentracing/ext/tags.py]: @oibe sure: https://github.com/opentracing/specification/blob/master/semantic_conventions.md

Comment (bhs) [inline opentracing/ext/tags.py]: (or https://github.com/opentracing/specification/blob/master/semantic_conventions.yaml)

**Comment by oibe** [comment]
addressed in https://github.com/opentracing/opentracing-python/pull/42

**Pull request closed without merging**